Domain Connect Spec Draft.adoc

```diff
@@ -555,11 +582,6 @@ template for a service that is sold through different companies. Not all templat
 |groupId
 |This OPTIONAL parameter specifies the group of changes from the template to apply. If no group is specified, all groups are applied. Multiple groups can be specified in comma delimited format.
 
-|*Provider Name*

```

Comment:
Do we want to drop it? Wasn't that needed for Reseller case?

Reply:
This was in the table twice.  It is still in there :-)

Reply:
:ok: